arrayMethodsLab.js

```diff
@@ -0,0 +1,54 @@
+// 1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
+
+const upTen = (arr) => arr.map((el) => el += 10 )
+
+console.log(upTen([1,2,3]))
+
+// 2. Given an array, return a new array where every non-string element is converted to an empty string.
+
+const emptyString = (arr) => {
+  return arr.map((el) => {
+    if (typeof el !== typeof "a") {
```

Comment:
For future reference, typeof's return is unusual, it always returns a string of the type. Ex. typeof 5 -> 'number'